include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -43,6 +31,30 @@ struct MoteusJointConfig{
 class JointMoteus: public JointBase
 {
     public:
+        /**
+         * @brief Define a default resolution struct
+         * 
+         */
+        static const ::mjbots::moteus::QueryCommand kDefaultQuery;
+
+        /**
+         * @brief Define a resolution struct that includes torques
+         * 
+         */
+        static const ::mjbots::moteus::QueryCommand kTorqueQuery;
+
+        /**
+         * @brief Define a debug resolution struct
```

Comment:
What is the difference between `kDebugQuery` and `kComprehensiveQuery`? Its not obvious from the comments.